### PR TITLE
Add AllMySms channel.

### PR DESCRIPTION
**Pull request by mikaelpopowicz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,123 @@
...
+This method needs to return a phone number.
+
+```php
+public function routeNotificationForAllMySms(Notification $notification)
+{
+    return $this->phone_number;
+}
+```
+
+For testing purpose you may set the `ALL_MY_SMS_UNIVERSAL_TO` environment variable to always send SMS to the configured number.
```
Comment (atymic) [inline README.md]: I'd prefer if this was renamed to indicate it was for testing :)

File: src/AllMySmsMessage.php [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        return $this;
+    }
+
+    /**
+     * Set the sender name the message should sent from.
+     *
+     * @param  string  $sender
+     * @return $this
+     */
+    public function sender(string $sender)
```
Comment (atymic) [inline src/AllMySmsMessage.php]: maybe worth adding a return type of `self`?

File: src/AllMySmsMessage.php [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    /**
+     * Set the date the message should sent at.
+     *
+     * @param  \DateTime|string  $sendAt
+     * @return $this
+     * @throws \Exception
+     */
+    public function sendAt($sendAt)
+    {
+        if (! $sendAt instanceof DateTime) {
```
Comment (atymic) [inline src/AllMySmsMessage.php]: Can we use `DateTimeInterface` to support the immutable variants please

**Comment by atymic** [comment]
Repo is created.
https://github.com/laravel-notification-channels/all-my-sms

Please PR code there for final review, and see the launch checklist
https://github.com/laravel-notification-channels/all-my-sms/issues/1

File: README.md (hunk 1 above)
Comment (mikaelpopowicz) [reply README.md]: Maybe I should mention `Local development` like the Laravel mail documentation : https://laravel.com/docs/6.x/mail#mail-and-local-development

File: src/AllMySmsMessage.php (hunk 2 above)
Comment (mikaelpopowicz) [reply src/AllMySmsMessage.php]: In the method signature or the doc block ?

File: src/AllMySmsMessage.php (hunk 3 above)
Comment (mikaelpopowicz) [reply src/AllMySmsMessage.php]: Yep

**Pull request closed without merging**